### DISCUSSION TITLE
I Moved the folder....

### DISCUSSION THREAD

**Discussion by ShaoIndie** [V1P3R-FMG/free-map-genie#52] [Q&A]
So i moved the folder of the extension, somewhere else on my pc, and now that i opened my interactive map, it didnt load anything....

I read that the data should be saved locally by the browser... but...  Is  it somewhere on my computer? Can I import it manually?

**Comment by V1P3R-FMG** [comment]
Data is stored in localstorage in the browser, so as long as you don't switch or clear browser data your data should be fine.
So moving the extension folder shouldn't be a problem just make sure that your browser still knows the extensions path by loading it again as developer extension.

If you go to a given mapgenie map you can use the extension popup to import / export mapdata.
![image](https://github.com/V1P3R-FMG/free-map-genie/assets/114585881/42726349-88bc-40d4-a856-3fd873e9baab)

You can also find your stored data by opening the developer tools on any mapgenie map.
and looking for keys that start with `fmg` in localstorage.
![image](https://github.com/V1P3R-FMG/free-map-genie/assets/114585881/89dd9d78-9806-40c2-b601-cb760e1b5c85)
